api/pom.xml

```diff
@@ -15,20 +15,45 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
 			    http://maven.apache.org/maven-v4_0_0.xsd">
+
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
-        <version>1.0.6</version>
+        <version>1.0.7</version>
     </parent>
+
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.sun.activation</groupId>
-    <artifactId>all</artifactId>
-    <packaging>pom</packaging>
+    <groupId>jakarta.activation</groupId>
+    <artifactId>jakarta.activation-api</artifactId>
     <version>2.1.0-SNAPSHOT</version>
-    <name>Jakarta Activation distribution</name>
-    <description>${project.name}</description>
+    <packaging>jar</packaging>
+    <name>Jakarta Activation API</name>
     <url>https://github.com/eclipse-ee4j/jaf</url>
 
+    <properties>
+        <spec.version>2.0</spec.version>
+        <spec.title>Jakarta Activation Specification</spec.title>
+        <activation.recompile.skip>false</activation.recompile.skip>
+        <!-- for the osgiversion-maven-plugin -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <javadoc.title>Jakarta Activation API documentation</javadoc.title>
+        <copyright.exclude>${project.basedir}/../etc/copyright-exclude</copyright.exclude>
+        <copyright.ignoreyear>false</copyright.ignoreyear>
+        <copyright.scmonly>true</copyright.scmonly>
+        <copyright.update>false</copyright.update>
+        <spotbugs.skip>true</spotbugs.skip>
+        <spotbugs.threshold>High</spotbugs.threshold>
+        <!--Maven plugins version-->
+        <spotbugs.plugin.version>4.2.0</spotbugs.plugin.version>
+
+        <findbugs.skip>
+            false
+        </findbugs.skip>
+        <findbugs.exclude>
+            ${project.basedir}/exclude.xml
+        </findbugs.exclude>
```

Comment:
I'd collapse the two props above to one line/property; no reason for one prop to occupy 3 lines

Reply:
Done